Options/Applicative.hs

```diff
@@ -14,6 +15,11 @@ module Options.Applicative (
   -- | The standard @Applicative@ module is re-exported here for convenience.
   module Control.Applicative,
 
+  -- | Re-export Data.Monoid's (<>) operator for convenience.
```

Comment:
> That said, thank you both for the input. I'll probably drop the export completely and let the user decide which instance to use.

As a frequent user of the library I think that is a pretty good way forward. I often end up hiding `(<>)` from `Options.Applicative` anyways.